src/devices/ledvance.ts

```diff
@@ -332,6 +332,15 @@ const definitions: DefinitionWithExtend[] = [
         description: 'SMART+ Outdoor Plug EU',
         extend: [ledvanceOnOff({powerOnBehavior: false})],
     },
+    {
+        zigbeeModel: ['PLUG COMPACT EU EM T'],
+        model: 'PLUG COMPACT EU EM T',
+        vendor: 'LEDVANCE',
+        description: 'Automatically generated definition',
```

Comment:
Fill in a sensible description here.

Reply:
Description should be : PLUG COMPACT EU EM T